src/dependencyresolver.js

```diff
@@ -85,6 +85,8 @@ class DependencyResolver extends events.EventEmitter {
   async _pollResolvedTasks() {
     while(!this._stopping) {
       let messages = await this.queueService.pollResolvedQueue();
+      this.monitor.count('resolved-queue-poll-requests', 1);
```

Comment:
just a fyi, but I think saying you're counting by one is superfluous here, I think it automatically uses one